lib/src/main/java/com/auth0/msg/Message.java

```diff
@@ -0,0 +1,87 @@
+package com.auth0.msg;
```

Comment:
In abstract message, i have suggested some changes to methods, please make the same changes to interface.

Reply:
done